### PR TITLE
Add manual for long jobs

### PR DESCRIPTION
**Pull request by jelletreep**

### CODE REVIEW

File: docs/manuals/long-jobs.qmd [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+```
+
+After submitting this command you will see the following:
+
+![](../imgs/background.png)
+
+You need to press Enter one more time to return to the command prompt.
+The number between brackets is the job id and the second number is the process id.
+
+### Terminating the process
```
Comment (dometto) [inline docs/manuals/long-jobs.qmd:55]: Maybe add a subsection "how do I know whether my job has completed?"

File: docs/manuals/long-jobs.qmd [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- In Jupyterhub (Jupyter Workspace in Research cloud): 
+  To open a new terminal, click the + button in the file browser and select the terminal in the new Launcher tab
+  ([find a short video here](https://jupyterlab.readthedocs.io/en/stable/user/terminal.html)).
+- In Rstudio (Rstudio workspace in Research cloud):
+  In the bottom left panel, click the 'terminal' tab.
+- In an Ubuntu Desktop workspace, click 'Applications' in the top left corner and then 'Terminal'
+
+### Step 2: Navigate to the directory where your script is located
+```
+cd data
```
Comment (dometto) [inline docs/manuals/long-jobs.qmd:29]: Maybe add "For example" to not confuse users who are new to the terminal (I mean you don't always need to `cd data` :))

File: docs/manuals/long-jobs.qmd [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+- When you access your workspace via your web browser (Jupyter Notebooks, RStudio, Ubuntu Desktop, etc.)
+- When you access your workspace via SSH
+
+[More info on access methods](https://utrechtuniversity.github.io/vre-docs/docs/first-steps.html#getting-access-to-your-workspace)
+
+## How to run jobs as background processes
+
+### Prerequisites
+- You will need some basic linux command line skills to be able to run scripts as background processes. If you don't have these skills, take some time to practice using sections 1, 2, 3 and 7 of this short [online course](https://swcarpentry.github.io/shell-novice) before proceeding. You can practice in the terminal (see step 1).
+- Your script should be 'standalone'; which means that you should be able to run your entire script in one go without modifications. All figures and output data should be stored in files (e.g. `.png` images and/or `.csv` tables, this is not strictly necessary for jupyter notebooks).
```
Comment (dometto) [inline docs/manuals/long-jobs.qmd:17]: > without modifications

Maybe: "without providing additional input" (or: interactive input)

> this is not strictly necessary for jupyter notebooks

I found this a little bit confusing; why not?

**Review by dometto — COMMENTED** [review]
Great addition, this will be very helpful to our users! Added some suggestions in the comments.

**Comment by jelletreep** [comment]
Thanks for your suggestions! I made changes based on your suggestions

**Pull request merged**